xff/xff.go

```diff
@@ -118,8 +118,8 @@ func ParseTrustedProxies() []string {
 	return result
 }
 
-// New returns a set of grpc interceptors that will replace peer.IP with X-FORWARDED-FOR value if the peer ip within a subnet in allowedSubnets
-// If allowedSubnets is nil it will look for subents in the TRUSTED_PROXIES env var.
+// GRPCMiddlewares returns a set of grpc interceptors that will replace peer.IP with X-FORWARDED-FOR value if the peer ip within a subnet in allowedSubnets
```

Comment:
Oh I just noticed some grammatical issue here from before, can you fix though? Maybe to read like:
> if the peer IP is within one of the subents in allowedSubnets

Reply:
fixed: https://github.com/tinkerbell/hegel/pull/27/commits/884567ec2be5bdd2c541817fee72ec711a33fabb#diff-48a11bae66acafe529b7385a85a6a9e9R121